doc/alertmanager-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,310 @@
+        )
+        .addTarget(prometheus.target('sum(rate(alertmanager_alerts_received_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m]))' % $._config, legendFormat='Received'))
+        .addTarget(prometheus.target('sum(rate(alertmanager_alerts_invalid_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m]))' % $._config, legendFormat='Invalid'));
+
+      local notifications =
+        graphPanel.new(
+          'Rate',
+          datasource='$datasource',
+          span=4,
+          format='ops',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target('sum(rate(alertmanager_notifications_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])) by (integration)' % $._config, legendFormat='{{integration}}'));
+
+      local notificationErrors =
+        graphPanel.new(
+          'Error',
+          datasource='$datasource',
+          span=4,
+          format='ops',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target('sum(rate(alertmanager_notifications_failed_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])) by (integration)' % $._config, legendFormat='{{integration}}'));
+
+      local notificationDuration =
+        graphPanel.new(
+          'Duration',
+          datasource='$datasource',
+          span=4,
+          format='s',
+          stack=false,
+          fill=1,
+          legend_show=true,
+        )
+        .addTarget(prometheus.target(
+          |||
+            histogram_quantile(0.99,
+              rate(alertmanager_notification_latency_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
+          ||| % $._config, legendFormat='{{integration}} - 99th Percentile'
+        ))
+        .addTarget(prometheus.target(
+          |||
+            histogram_quantile(0.50,
+              rate(alertmanager_notification_latency_seconds_bucket{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            ) 
+          ||| % $._config, legendFormat='{{integration}} - Median'
+        ))
+        .addTarget(prometheus.target(
+          |||
+            rate(alertmanager_notification_latency_seconds_sum{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+            /
+            rate(alertmanager_notification_latency_seconds_count{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m])
+          ||| % $._config, legendFormat='{{integration}} - Average'
+        ));
+
+
+      local silences =
+        graphPanel.new(
+          'Silences',
+          datasource='$datasource',
+          span=4,
+          format='none',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('sum(alertmanager_silences{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}) by (state)' % $._config, legendFormat='{{ state }}'));
```

Comment:
Each instance should report the same number of silences. So again, summing them up will artificially inflate the number. You could again break up by instance, as different silence counts signal a problem.